app/assets/stylesheets/components/_navbar.scss

```diff
@@ -51,9 +51,10 @@ input {
 }
 
 .navbar-troveko-sticky {
-  position: relative;
-  z-index:1000;
-  width:100%;
+  position: sticky;
```

Comment:
`position: sticky` has very poor browser support. check out [the feature's caniuse page](https://caniuse.com/#search=sticky)